metricbeat/docs/developer-guide/metricset-details.asciidoc

```diff
@@ -208,6 +209,11 @@ To run all the tests, run `make testsuite`. To only run unit tests, run
 `make unit-tests` or for integration tests `make integration-tests-environment`. Be aware that
 a running Docker environment is needed for integration and system tests.
 
+Sometimes you may just want to run specify single integration test, such as `apache` module, firstly, start the docker service by run
```

Comment:
Try to split this up into multiple sentence or perhaps a bullet list. @dedemorton has perhaps some recommendations here?